config/splunk_matrix.conf

```diff
@@ -1,7 +1,17 @@
 [GENERAL]
-LATEST = 8.2
+LATEST = 9.0
 OLDEST = 8.0
 
+[9.0]
+VERSION = 9.0.0
+BUILD = 6818ac46f2ec
+SUPPORTED = 2024-06-14
+PYTHON3 = true
+PYTHON2 = true
```

Comment:
What is the usecase of this? As Splunk 9 drops support for py2 i think

Reply:
Changed to false. I believe (by quick code investigation) this parameter might not be used at all